libsrc/lynx/cgetc.s

```diff
@@ -52,6 +54,8 @@ _cgetc:
         rts
 @L5:
         lda     KBEDG           ; No Pause pressed
+        ldx     #$00
```

Comment:
Can these 2 lines be replaced with "stz KBEDG" ?

Reply:
I think so. Thanks! I am poor in Assembly and this is an extra opcode not available on standard 6502.

@BrianPeek Could you please test this fix?

Reply:
@BrianPeek I have pushed the the fix with STZ

Reply:
Did a very quick compile of the lynx lib in your branch/fork and it seems to be working correctly.  I will verify again when the PR is accepted into the main branch and the Windows snapshot is available.  But, overall, this seems good.  Thanks for the fix!